Gruntfile.js

```diff
@@ -33,6 +85,7 @@ grunt.registerTask( "update-authors", function() {
 	});
 });
 
-grunt.registerTask( "default", [ "jshint", "jscs" ] );
+grunt.registerTask( "default", [ "watch" ] );
```

Comment:
I would not have watch run on the default task I never ever use watch. Im all for having a watch task just not as the default.